hooks/cache_location.py

```diff
@@ -98,12 +98,14 @@ def _get_cache_root(self, project_id, pipeline_configuration_id):
 
         # get site only; https://www.foo.com:8080 -> www.foo.com
         base_url = urlparse.urlparse(tk.shotgun.base_url)[1].split(":")[0]
-        
+
         # now structure things by site, project id, and pipeline config id
-        cache_root = os.path.join(root, 
-                                  base_url, 
-                                  "project_%d" % project_id,
-                                  "config_%d" % pipeline_configuration_id)
+        cache_root = os.path.join(
```

Comment:
will this break for clients who have a shared site core and a cache hook? Super edge case, but just checking...